crypto/doc/CHANGELOG_oberon.rst

```diff
@@ -48,12 +48,12 @@ Removed
 
 Library built against Mbed TLS version 3.3.0.
 
-Added the following Oberon crypto libraries for nRF9160, nRF53, nRF52, and nRF51 architectures.
+Added the following Oberon crypto libraries for nRF91, nRF53, nRF52, and nRF51 architectures.
```

Comment:
Hmm. Wonder if since these are all also nRFxx Series, there should just be a Series at the very end.